update for block changes in 5.3

joyously opened the pull request:
Some block syntax changed and some inline styles changed to class names.
Added missing core blocks.
For #55

carolinan commented:
I like the text changes a lot, and the block additions.
But I am getting 404 messages for most of the images in the block gallery. 
Not for the classic gallery.

joyously commented:
I updated it with the original image paths (which are of course the old date). Funny that just this morning, we were discussing in Slack #forums a forum question about which date is used on image uploads. New images for an old post use the post publish date. It seems that imports use the post publish date also, so I changed those to match the original, just so everything would match. It seems that the importer doesn't deal with all URLs in blocks. The `data-link` is ignored.
See https://github.com/WordPress/wordpress-importer/issues/63
_So after import, there are still references to the original site (links and File block)._
